src/main/java/io/ebean/dbmigration/ddlgeneration/platform/BaseTableDdl.java

```diff
@@ -715,7 +729,7 @@ public void generate(DdlWrite writer, AddColumn addColumn) throws IOException {
       alterTableAddColumn(writer.apply(), tableName, column, false);
     }
 
-    if (isTrue(addColumn.isWithHistory())) {
+    if (isTrue(addColumn.isWithHistory()) && !sql2011History) {
```

Comment:
as far as I understand, when the db provides true sql2011 history-support, we must not touch the *_history tables.
(at least SqlSever adds new colums automatically to the history table)